# ecoli_pangenome
Pangenome analysis of Escherichia coli genomes

Background: 
Escherichia coli (E. coli) has been one of the most studied model organism in the history of life sciences. Initially thought just to be commensal bacteria, E. coli strains have shown wide phenotypic diversity including pathogenic isolates with great relevance to public health. Though pangenome analysis of E. coli has been attempted several times, however, there is no systematic functional characterization of the E. coli subgroups according to the gene profile. 

Results:
We have systematically built the E. coli pan-genome from the list of 1,324 complete genomes. The pangenome size is estimated to be ~25,000 gene families. Whereas the core genome diminishes as more genomes are added, the softcore genome (â‰¥95% of strains) is stable with ~3,000 gene families regardless of the total number of genomes. Unsupervised clustering of common E. coli sequence types using the presence/absence gene family matrix reveals distinct characteristics of E. coli phylogroup B1, B2 and E.  Among other observations, we highlight the bi-lineage nature of E. coli phylogroup B1, the variation of secretion systems and of the iron acquisition system in E. coli ST11 and the incorporation of highly conserved prophage genome of E. coli ST131 strains. The tail structure of the prophage is evolutionarily related to R2-pyocin (a tailocin) from Pseudomonas aeruginosa PAO1. We hypothesize that this molecular machinery is highly likely to play an important role in the lytic cycle protecting the own colonies; thus, potentially contributing towards the rapid rise of pandemic E. coli ST131.

Conclusion:
This study has explored the systematic pangenome development in bacterial species (specifically E. coli) by computational parametrization. We provide complete gene family lists of the E. coli pangenome as supplementary data for further studies. We also applied the pangenome matrix for identifying important biological characteristics of different E. coli subtypes, specifically phylogroups B1, B2 and E. We find an operon-like genome region that is highly specific for ST131 strains which could act as a potential killer weapon that provides E. coli ST131 strains with a competitive advantage with other closely related bacteria and, suggestively, explains their dominance as human pathogen among E. coli strains. 

Supplementary Data:
The data accompanied this study has been put in the folder "pangenome-results". 
The protein sequences can be obtained from the folder "protein-sequences".
The scripts, libraries and detail steps on the analysis can be obtained from the folder "supplementary-methods".
